admin/src/components/util/helpers/form-errors-helper.spec.js

```diff
@@ -0,0 +1,122 @@
+describe("Form validations", () => {
+  it("It validates new tag", () => {
+    cy.visit("/tag/new");
```

Comment:
Test of form error helper requires that `/tag/new` exits (hidden dependency) can we do testing without this dependency?